.github/workflows/pr-to-discord.yml

```
@@ -0,0 +1,68 @@
+        id: branch
+        run: |
+          if [ "${{ github.event.pull_request.base.ref }}" != "dev" ]; then
+            echo "Not dev branch, skipping"
+            exit 0
+          fi
+
+      - name: Send Discord notification
+        if: github.event.action != 'closed'
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)